aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/customize/s3/S3ExpressDecorator.kt

```diff
@@ -80,6 +83,41 @@ class S3ExpressDecorator : ClientCodegenDecorator {
             S3ExpressRequestChecksumCustomization(
                 codegenContext, operation,
             )
+
+    override fun extraSections(codegenContext: ClientCodegenContext): List<AdHocCustomization> {
+        return listOf(
+            adhocCustomization<SdkConfigSection.CopySdkConfigToClientConfig> { section ->
+                rust(
+                    """
+                    ${section.serviceConfigBuilder}.set_disable_s3_express_session_auth(
+                        ${section.sdkConfig}
+                            .service_config()
+                            .and_then(|conf| {
+                                let str_config = conf.load_config(service_config_key("AWS_S3_DISABLE_EXPRESS_SESSION_AUTH", "s3_disable_express_session_auth"));
+                                str_config.and_then(|it| it.parse::<bool>().ok())
+                            }),
+                    );
+                    ${section.serviceConfigBuilder}.set_disable_multi_region_access_points(
```

Comment:
Thanks for taking care of S3 Express related configs! Is `disable_multi_region_access_points` supposed to be configured from `S3ExpressDecorator` as opposed to `S3Decorator`?